gson/src/main/java/com/google/gson/stream/JsonReader.java

```diff
@@ -1505,7 +1551,7 @@ private String getPath(boolean usePreviousPath) {
    *   <li>For JSON arrays the path points to the index of the previous element.<br>
    *   If no element has been consumed yet it uses the index 0 (even if there are no elements).</li>
    *   <li>For JSON objects the path points to the last property, or to the current
-   *   property if its value has not been consumed yet.</li>
+   *   property if its name has already been consumed.</li>
```

Comment:
The previous documentation (introduced in my PR #2000) might have been a bit misleading because whether or not the value has been consumed does not make a difference. (It might only affect which property you consider the "last" and the "previous".)